src/blue-theme.scss

```diff
@@ -0,0 +1,7 @@
+@charset "utf-8";
+$democracy-blue: #2e4272;
```

Comment:
Maybe we should use a different colour?
There is not enough contrast between the logo colour and the background colour.
![image](https://user-images.githubusercontent.com/2419963/82301411-83a6cb80-99b8-11ea-8a77-21cfdf9c409b.png)

The blue is so dark it appears practically black for elements with thin borders:
![image](https://user-images.githubusercontent.com/2419963/82301556-b6e95a80-99b8-11ea-931c-98062f05d9b3.png)

Reply:
![Lavinia-theme](https://user-images.githubusercontent.com/23212478/82927368-eb44b400-9f80-11ea-8f25-f750f1d0b922.png)

With a white logo it looks ok I think.

Reply:
![light-blue-theme](https://user-images.githubusercontent.com/23212478/82930153-38c32000-9f85-11ea-98c5-e91d49c99770.png)

Reply:
![Lavinia-theme](https://user-images.githubusercontent.com/23212478/82930951-770d0f00-9f86-11ea-9dd0-ac89a67d3c53.png)

Reply:
It looks a lot better with the white logo, the light blue is a bit too bright.
What do you think of this colour? #1340bd

Reply:
![blue_#1340bd](https://user-images.githubusercontent.com/23212478/82939038-00c2d980-9f93-11ea-8ae8-cede0134774f.png)

A bit intense maybe

Reply:
Let's just go for the original colour and get this PR through.

Reply:
Alright